src/app/design/frontend/base/default/template/shopgate/cloudapi/pipelineRequest.phtml

```diff
@@ -3,4 +3,4 @@
     document.observe("dom:loaded", function () {
         ShopgatePipelineRequest.fire(<?php echo $this->getJsonMethods() ?>)
     });
-</script>
\ No newline at end of file
+</script>
```

Comment:
@augsteyer I double checked newlines with @awesselburg seems that they are not displayed at github. But there will be an icon (see one line above) in case there is no newline at the end of the file. So should be fine for this PR now.